python_modules/dagster/dagster/_core/remote_representation/external_data.py

```diff
@@ -1748,7 +1750,7 @@ def external_asset_nodes_from_defs(
             op_names = sorted([str(handle) for handle in node_handles])
             op_name = graph_name or next(iter(op_names), None) or node_def.name
             job_names = sorted([jd.name for jd in job_defs_by_asset_key[key]])
-            compute_kind = node_def.tags.get("kind")
```

Comment:
afaict this is the only use of the op tag, but might be worth pulling this into some sort of util to unify op/asset versions to pre-empt any dagster lib code dependent on compute kind outside of the UI (search, views, asset selection logic etc)